src/libmongoc/doc/installing.rst

```diff
@@ -251,17 +252,23 @@ Let's start by generating Visual Studio project files. The following assumes we
 
 (Run ``cmake -LH ..`` for a list of other options.)
 
-Now that we have project files generated, we can either open the project in Visual Studio or compile from the command line. Let's build using the command line program ``msbuild.exe``:
+To see a complete list the CMake generators available on your specific system, use a command like this:
+
+.. parsed-literal::
+
+ $ cmake -E capabilities | json_pp -json_opt pretty,canonical
```

Comment:
Can we replace this with `cmake --help` instead? At least on macOS that lists available generators.

Reply:
Done.